cms/settings.py

```diff
@@ -129,6 +129,7 @@
 USERS_CAN_SELF_REGISTER = True
 
 RESTRICTED_DOMAINS_FOR_USER_REGISTRATION = ["xxx.com", "emaildomainwhatever.com"]
+ALLOWED_DOMAINS_FOR_USER_REGISTRATION = []
```

Comment:
add a comment, eg 
# comma separated list of domains, eg ['domain1.org', 'domain2.org']

Reply:
Complete.